docs/rcait.md

```diff
@@ -77,6 +117,12 @@ framework, will re-run them for the i+1 the iteration till a timeout. The timeou
 configurable to any number of seconds using the field `timeoutSeconds` but has the default
 of 60 seconds. 
     
+__AErrorPatternIgnored@__
```

Comment:
I wanted to understand why you chose to use `@` towards the end of decorator name rather than front like others ?

Reply:
Good catch !